File: apps/ResidentApp.Android/app/src/main/java/com/acls/resident/data/remote/dto/OutageDtos.kt
package com.acls.resident.data.remote.dto

import com.google.gson.annotations.SerializedName

data class OutageDto(
    @SerializedName("outageId") val outageId: Int,
    @SerializedName("propertyId") val propertyId: Int,
    @SerializedName("title") val title: String,
    @SerializedName("outageType") val outageType: String,
    @SerializedName("description") val description: String,
    @SerializedName("startTime") val startTime: String,
    @SerializedName("endTime") val endTime: String,
    @SerializedName("declaredAt") val declaredAt: String,
    @SerializedName("notificationSentAt") val notificationSentAt: String?
)

File: apps/ResidentApp.iOS/Sources/OutageDTOs.swift
import Foundation

struct OutageDTO: Codable, Identifiable {
    let outageId: Int
    let title: String
    let outageType: OutageType
    let description: String
    let startTime: Date
    let endTime: Date
    let declaredAt: Date
    let notificationSentAt: Date?

    var id: Int { outageId }
}
